app/src/main/java/org/fossasia/openevent/general/event/EventsViewModel.kt

```diff
@@ -60,6 +74,12 @@ class EventsViewModel(
         } else {
             mutableProgress.value = false
         }
+
+        mutableProgress.observeForever {
```

Comment:
@iamareebjamal I can't observer in the ViewModel i guess. can i ?

Reply:
No, use MediatorLiveData

Reply:
@iamareebjamal updated

Reply:
@iamareebjamal should I implement pagination for search events as well?

Reply:
Build is failing. Yeah, implement that in other PR

Reply:
There are some unrelated changes will undo them and fix spotless. Rest all good?